torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -502,6 +502,83 @@ def _sfdp_replacement_18(query, key, value, causal_mask, dropout_p):
     )
 
 
+def _sfdp_pattern_19(
+    query,
+    key,
+    value,
+    attn_mask,
+    inv_scale,
+    q_zp,
+    q_scale,
+    k_zp,
+    k_scale,
+    v_zp,
+    v_scale,
+    a_zp,
+    a_scale,
+    a_inv_scale,
+    o_zp,
+    o_inv_scale,
+    dropout_p,
+):
+    # UINT8 QUANTIZED SDPA
+    q = query.permute([0, 2, 1, 3])
+    q = (q.float() - q_zp) * q_scale
+    k = key.permute([0, 2, 1, 3]).transpose(-2, -1)
+    k = (k.float() - k_zp) * k_scale
+    v = value.permute([0, 2, 1, 3])
+    v = (v.float() - v_zp) * v_scale
```

Comment:
Since the fp32 pattern is just a sub pattern of this, why can't we just match the fp32 pattern?

Reply:
After the support of UINT8 SDPA kernel in the future, we could directly call UINT8 SDPA in replacement instead of doing quant and dequant. Here, we want to match a larger scope of pattern than fp32, in order to get UINT8 as input type.

Reply:
But why do we need this intermediate PR?

Reply:
Just want to do some works before the kernel is ready. It is OK to add the corresponding patterns after the kernel implemented, and I will pend this for now. Thanks!

Reply:
Hi, @jgong5 & @Valentine233, would it be okay to land this pattern by disabling its replacement,?
Having this serialized pattern already in-tree would help me use it in #126740. Thanks!